src/haz3lcore/dynamics/Builtins.re

```diff
@@ -75,16 +110,51 @@ module Pervasives = {
         raise(EvaluatorError.Exception(InvalidBoxedFloatLit(d1)))
       | Indet(d1) => Indet(ApBuiltin(name, [d1])) |> return
       };
+
+    /* float_of_string implementation. */
+    let float_of_string = (name, r1) =>
+      switch (r1) {
+      | BoxedValue(StringLit(s) as d1) =>
+        let s = Re.Str.string_after(s, 1);
```

Comment:
hm, is this because the argument to StringLit includes the quotation marks?

Reply:
It is because of that, yes.
I coped it to mirror how int_of_string handled it, but could look into modifying both to handle it a different way if that would be better